saleor/search/backends/base.py

```diff
@@ -227,8 +235,10 @@ def add_bulk(self, model, obj_list):
     def delete(self, obj):
         raise NotImplementedError
 
-    def search(self, query_string, model_or_queryset, fields=None, filters=None,
-               prefetch_related=None, operator=None, order_by_relevance=True):
+    def search(self, query_string, model_or_queryset, fields=None,
+               filters=None,
+               prefetch_related=None, operator=None, order_by_relevance=True,
+               page=1, page_size=16):
```

Comment:
I think you may want to use constant `PAGINATE_BY` or `DASHBOARD_PAGINATE_BY`

Reply:
Looking at this again I think we don't need default values here - if none provided then ES uses its own pagination.